scripts/onboarding/managed/enable-monitoring.sh

```diff
@@ -340,7 +340,7 @@ validate_cluster_identity() {
   local clusterName="$(echo ${2})"
 
   local identitytype=$(az resource show -g ${rgName} -n ${clusterName} --resource-type $resourceProvider --query identity.type)
-  identitytype=$(echo $identitytype | tr "[:upper:]" "[:lower:]" | tr -d '"')
+  identitytype=$(echo $identitytype | tr "[:upper:]" "[:lower:]" | tr -d '"' | tr -d "[:space:]")
```

Comment:
This check was added for private preview of Arc K8s. Since Arc K8s GA, identitytype check not required any more. Can you please help on removing this completely? And also, we are working release monitoring extension and that time this script not used any more.